packages/orchestration/src/orchestration-types.ts

```diff
@@ -31,6 +31,18 @@ export type LlmModuleConfig = OriginalLlmModuleConfig & {
   model_name: ChatModel;
 };
```

Comment:
```suggestion
model_params?: LlmModelParams;
};
```

How about if we do this? The user won't need to import another function.

Reply:
Yeah I thought of this at first but i thought we would want a more narrow suggestion for the user. I'm okay with this as well.

Reply:
I would prefer this (more intuitive, no issue with discoverability). @tomfrenken WDYT?

Reply:
Fine with me as well, since we assume that these parameters are essentially always available, it makes sense to have it here too. We need to be aware though, that we are overriding the OriginalLlmModuleConfigs's model_param type then, right now it is Record<string,any>, it could be more narrow in the future though.